site/contents/write_ups/2021/11-08-21_writeup.md

```diff
@@ -0,0 +1,119 @@
+
+The media also platform anti-climate change "scientists" and spokespeople. For example [Boris Johnson's father being interviewed as an environmentalist by the BBC](https://www.thenational.scot/news/19504423.bbc-newsnight-greens-appalled-stanley-johnsons-appearance/) - there is inherent bias in the media. 
+
+This idea of "equal debate" is flawed. If 10% of people believe in one thing and 90% don't, giving representives from both viewpoints equal airtime is not "equal". This has been bought to light during the pandemic, where we see qualified epidemeologists debating youtubers.
+
+What constitutes an expert? It's usually seen as negative when people change their minds, but acually that's a reflection of changing evidence (i.e. science). An interesting exploration on this tangent can be found in this [video](https://www.youtube.com/watch?v=LRNkDZy30xU).
+
+On the other hand, we can be more positive about our ability to affect change. 
+
+As Gerry McGovern notes in his book [World Wide Waste](https://gerrymcgovern.com/world-wide-waste/), all of our digital actions have physical consequences:
+
+- 1.6 billion trees would have to be planted to offset the pollution caused by email spam.
+- 1.5 billion trees would need to be planted to deal with annual e-commerce returns in the US alone. 
+- 231 million trees would need to be planted to deal with the pollution caused as a result of the data US citizens consumed in 2019.
+- 16 million trees would need to be planted to offset the pollution caused by the estimated 1.9 trillion yearly searches on Google. 
+
+Maybe, as Charles Radclyffe suggests in his piece [The Dirty Secret About AI](https://www.forbes.com/sites/charlesradclyffe/2019/06/10/ais-dirty-secret/?sh=10b56de65331), the solution could very well be switching off AI.
+
+If these seem a bit extreme, consider how, with little to no policy changes plastic straws were all but phased out in cafes and coffee shops, as well as a mass decrease in plastic bag sales from supermarkets. There is precident for large change being affected without policy makers enforcing it.
+
+We need to change the personal norms and our degree of excitement with new tech. We need to delay our renewal of our personal IT equipment (even delaying by 6 months the acquisition of our next phone might have an impact), we could shrink the size and number of files that we store in the cloud. 
+
+How do we shift the psychology around this? People feel that they lack agency and believe that they will struggle to make a change on their own. Leadership is needed. This doesn't have to be a policy maker. It just needs to be someone to get the message out about the impacts that our consumer decisions have.
```

Comment:
I feel like this line (89) belongs with line 72, about encouraging mass public action and what constitutes an expert and someone who we can trust. i.e. someone to be a leader, but also for us as a population to trust them and know that leaders changing their minds is not a sign of weakness but actually a sign of someone following the science.